asn1tools/codecs/ber.py

```diff
@@ -71,6 +71,9 @@ class Tag(object):
     DATE_TIME         = 0x21
 
 
+EOC_TAG = bytes(2)
```

Comment:
Hmmm, I think I prefer ``END_OF_CONTENTS_OCTETS`` to make it easier to understand